src/distdl/nn/conv_feature.py

```diff
@@ -206,6 +208,18 @@ def __init__(self,
         self._distdl_is_setup = False
         self._input_tensor_structure = TensorStructure()
 
+    def _expand_parameter(self, param):
+        # If the given input is not of size num_dimensions, expand it so.
+        # If not possible, raise an exception.
+        param = np.atleast_1d(param)
+        if len(param) == 1:
+            param = np.ones(self.num_dimensions) * param[0]
+        elif len(param) == self.num_dimensions:
+            pass
+        else:
+            raise ValueError('Invalid parameter: ' + str(param))
+        return param
```

Comment:
To match torch behavior, this should be a tuple.  Then the expansion can just be `[1]*n` and that list transformed to the tuple.